modules/home.php

```diff
@@ -1,7 +1,34 @@
 <div id="main">
     <div id="main-content">
-        <h3> The main content go here</h3>
+    <?php if ($user === false) { ?>
+        <h3>Welcome to the website! Login to see users list</h3>
+    <?php } else { ?>
+        <h3>Users List </h3>
+        <?php
+            try {
+                $result = $mysql->query($sql);
```

Comment:
You need to define SQL before execute it. 
If you don't define it here, mean that you are using sql defined in header.